llvm/labs/lab2/kruglov_alexey/MulToBitShift.cpp

```diff
@@ -0,0 +1,82 @@
+#include "llvm/IR/BasicBlock.h"
+#include "llvm/IR/Function.h"
+#include "llvm/IR/IRBuilder.h"
+#include "llvm/Pass.h"
+#include "llvm/Passes/PassBuilder.h"
+#include "llvm/Passes/PassPlugin.h"
+#include <stack>
+
+namespace {
+struct MulToBitShift : llvm::PassInfoMixin<MulToBitShift> {
+public:
+  llvm::PreservedAnalyses run(llvm::Function &F,
+                              llvm::FunctionAnalysisManager &FAM) {
+    for (llvm::BasicBlock &BB : F) {
+      std::stack<llvm::Instruction *> worklist;
+      for (llvm::Instruction &Inst : BB) {
+        if (Inst.getOpcode() == 17) { // Opcode for 'mul' Inst is 17

```

Comment:
Could you please use reserved enum value for comparison instead of magic number?